### ISSUE TITLE
Migration for noarch packages with pyXX-egg.info

### ISSUE THREAD

**Issue by isuruf** [regro/cf-scripts#797]
See https://github.com/conda/conda/issues/7693

If they are rebuilt with pip>9, then the correct directory structure will be created.

**Comment by isuruf** [comment]
If there are packages using  `--no-cache-dir` even with `pip>9`, this issue will be there. So, `--no-cache-dir` also needs to be removed.

**Comment by beckermr** [comment]
We have a piggyback for python build strings. Does this not cover it? 

I guess you want something explicit?

**Comment by isuruf** [comment]
We should look at packages built with older pip even without `--no-cache-dir`. Those wouldn't be in the migrator for python build strings.
I also think we should have a non-piggyback migrator to fix this.

**Comment by beckermr** [comment]
@regro/auto-tick @CJ-Wright where do we / can we pull in info from the about section of the built packages? This is where we can find what version of pip was used.

**Comment by CJ-Wright** [comment]
Libcfgraph

**Comment by isuruf** [comment]
You can also use `libcfgraph` to just check that there is a `pyXX-egg.info` directory instead of checking for the version of pip used.

**Comment by beckermr** [comment]
Right ok. I want to make sure that data is not already stored in the graph nodes in the migrator code (which are per feedstock and not output). I guess the answer there is no?

**Comment by CJ-Wright** [comment]
Yeah, the mapping between packages and feedstocks can be a bit rough
